packages/communication-react/CHANGELOG.md

```diff
@@ -9,45 +9,39 @@ This log was last generated on Mon, 24 Jan 2022 23:18:53 GMT and should not be m
 Mon, 24 Jan 2022 23:18:53 GMT 
 [Compare changes](https://github.com/azure/communication-ui-library/compare/1.0.1-beta.1...1.0.1-beta.2)
 
-### Minor changes
-
-- `@internal/react-components`
-  - Added strings to the side pane from locale context. ([PR #1278](https://github.com/azure/communication-ui-library/pull/1278) by 94866715+dmceachernmsft@users.noreply.github.com)
-- `@internal/react-composites`
-  - implemented custom datamodel functionality to meeting composite excluding sidebar. ([PR #1319](https://github.com/azure/communication-ui-library/pull/1319) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Added MeetingCompositeOptions type to the meetings composite. ([PR #1272](https://github.com/azure/communication-ui-library/pull/1272) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - changed meeting peopel pane to use ParticipantContainer Component. ([PR #1328](https://github.com/azure/communication-ui-library/pull/1328) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - added missing return types. ([PR #1332](https://github.com/azure/communication-ui-library/pull/1332) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Add API for injecting custom buttons into CallComposite ([PR #1314](https://github.com/azure/communication-ui-library/pull/1314) by 82062616+prprabhu-ms@users.noreply.github.com)
-  - Added strings to the side pane from locale context. ([PR #1278](https://github.com/azure/communication-ui-library/pull/1278) by 94866715+dmceachernmsft@users.noreply.github.com)
-  - Do not remove users from chat in the meeting composite. ([PR #1340](https://github.com/azure/communication-ui-library/pull/1340) by 2684369+JamesBurnside@users.noreply.github.com)
-- `@internal/storybook`
-  - fixed storybook controls around the meetings composite to work with the new MeetingCompositeOptions type in the MeetingCompositeProps ([PR #1272](https://github.com/azure/communication-ui-library/pull/1272) by 94866715+dmceachernmsft@users.noreply.github.com)
-
-### Patches
+### Changes
 
-- `@internal/calling-component-bindings`
+- Calling Component Bindings
```

Comment:
Note for future James or anyone who wants to dable with how we auto generate changelogs!, we should be able to have beachball generate these names as "Calling Component Bindings" etc. instead of the packlet name (`@internal/calling-component-bindings`).

The custom renders we currently have live here: https://github.com/Azure/communication-ui-library/blob/main/common/config/beachball/changelog-custom-renders.ts
And the variety of different renderers we can customize to get what we want can be found here: https://github.com/microsoft/beachball/blob/79367585d8bc2a33effa4b12705ba063b53d31ee/src/types/ChangelogOptions.ts#L69